SPECS/azurelinux-sysinfo/azurelinux-sysinfo.spec

```diff
@@ -38,6 +38,38 @@ install -m 755 %{SOURCE2} %{buildroot}%{_sysconfdir}/systemd/system/
 %{_sysconfdir}/systemd/system/azurelinux-sysinfo.service
 
 %post
+# Apply required SElinux policies only if SELinux mode is enforcing
+# and specified packages are present
+if rpm -q coreutils checkpolicy policycoreutils selinux-policy-devel &> /dev/null; then
+    if [[ $(getenforce) == "Enforcing" ]]; then
+        mkdir -p /tmp/
+        POLICYFILE=/tmp/sysinfo-selinuxpolicies.cil
+        cat << EOF > $POLICYFILE
```

Comment:
I don't necessarily know what best practice is here, but should this be a proper source file?